moncli/types.py

```diff
@@ -217,7 +180,79 @@ def _cast(self, value):
     def _export(self, value):
         return { 'rating': value}
 
-        
+
+class StatusType(MondayType):
+    native_type = str
+    allow_casts = (int,)
+    null_value = {}
+
+    def __init__(self, id: str = None, title: str = None, as_enum: type = None ,  *args, **kwargs):
+        if as_enum:
+            try:
+                enum_value = None
+                for value in list(as_enum):
+                    enum_value = value.name
+                    str(enum_value)
+                self._as_enum =  as_enum
+                self.native_type = Enum
+                self.allow_casts = str
+                self.choices = list(as_enum)
+            except TypeError:
+                raise ConversionError('Invalid value "{}" for status Enum "{}".'.format(enum_value, enum_value.__class__))    
+        super().__init__(id=id, title=title, *args, **kwargs)
+    
+    def _process(self, value):
+        labels = self.metadata['labels']
+        label = str(value)
+        if self.native_type != str:
+            return value
+        try:
+            value = int(value)
+            try:
+                return labels[label]
```

Comment:
You can simply replace _label_ here with _value_, as the _value_ will always be a str.